docs/source/executing-operations/demand-control.mdx

```diff
@@ -552,3 +386,192 @@ You can set alerts in your APM for events that may require changing your demand
+| `strategy`            | `static_estimated`      |     --          | `static_estimated` estimates the cost of an operation before it is sent to a subgraph                 |
+| `static_estimated.list_size`  | integer                 |     --          | The assumed maximum size of a list for fields that return lists.                                      |
+| `static_estimated.max`        | integer                 |     --          | The maximum cost of an accepted operation. An operation with a higher cost than this is rejected. |
+
+
+## Telemetry for demand control
+
+<Tip>
+
+New to router telemetry? See [Router Telemetry](../configuration/telemetry/overview).
+
+</Tip>
+
+You can define router telemetry to gather cost information and gain insights into the cost of operations sent to your router:
+
+- Generate histograms of operation costs by operation name, where the estimated cost is greater than an arbitrary value.
+- Attach cost information to spans.
+- Generate log messages whenever the cost delta between estimated and actual is greater than an arbitrary value.
+
+### Instruments
+
+| Instrument       | Description                                                |
+| ---------------- | ---------------------------------------------------------- |
+| `cost.actual`    | The actual cost of an operation, measured after execution.  |
+| `cost.estimated` | The estimated cost of an operation before execution.        |
+| `cost.delta`     | The difference between the actual and estimated cost.       |
+
+### Attributes
+
+Attributes for `cost` can be applied to instruments, spans, and events—anywhere `supergraph` attributes are used.
+
+| Attribute      | Value  | Description                                                |
+| --------------- | ----- | ---------------------------------------------------------- |
+| `cost.actual`     | boolean      | The actual cost of an operation, measured after execution.  | 
+| `cost.estimated`  | boolean      | The estimated cost of an operation before execution.        |
+| `cost.delta`      | boolean      | The difference between the actual and estimated cost.       |
+| `cost.result`     | boolean      | The return code of the cost calculation. `COST_OK` or an [error code](../errors/#demand-control)      |
+
+### Selectors
+
+Selectors for `cost` can be applied to instruments, spans, and events—anywhere `supergraph` attributes are used.
+
+| Key  | Value      | Default | Description                                        |
+| ---- | ---------- | ------- | -------------------------------------------------- |
+| `cost` | `estimated`, `actual`, `delta`, `result` |         | The estimated, actual, or delta cost values, or the result string |
+
+
+### Examples
+
+#### Example instrument
+
+Enable a `cost.estimated` instrument with the `cost.result` attribute:
+
+```yaml
+telemetry
+  instrumentation:
+    instruments:
+      supergraph:
+        cost.estimated: true
+	  attributes:
```

Comment:
@abernix do you know what this means? is it tabs vs spaces ?

Reply:
yes.  this is showing tabs. it should be spaces.

Reply:
i'm fixing.

Reply:
fixed yet? :D

Reply:
It was not easy, but it is fixed in 62a2a6d.